tempesta_fw/http_parser.c

```diff
@@ -383,7 +424,7 @@ enum {
  * (e.g. Content-Length which is doubled by TfwHttpMsg.conent_length)
  * to mangle row skb data.
  */
-#define TFW_HTTP_PARSE_SPECHDR_VAL(st_curr, st_i, hm, func, id)		\
+#define TFW_HTTP_PARSE_SPECHDR_VAL_GEN(st_curr, st_i, hm, func, id, saveval) \
```

Comment:
"GEN" is usually used for "generate", I'd prefer "__TFW_HTTP_PARSE_SPECHDR_VAL" for the new macro